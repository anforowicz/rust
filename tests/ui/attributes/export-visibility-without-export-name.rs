// This test verfies that `#[export_visibility = ...]` cannot be used without
// either `#[export_name = ...]` or `#[no_mangle]`.
#![feature(export_visibility)]
#[export_visibility = "hidden"]
pub static TESTED_STATIC: [u8; 6] = *b"foobar";

fn main() {}
