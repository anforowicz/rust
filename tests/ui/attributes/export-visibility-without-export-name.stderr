error: export visibility will be ignored without `export_name`, `no_mangle`, or similar attribute
  --> $DIR/export-visibility-without-export-name.rs:4:1
   |
LL | #[export_visibility = "hidden"]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

